Firmware/eeprom.cpp

```diff
@@ -140,15 +140,23 @@ bool __attribute__((noinline)) eeprom_is_sheet_initialized(uint8_t sheet_num) {
     return (eeprom_read_word(reinterpret_cast<uint16_t*>(&(EEPROM_Sheets_base->s[sheet_num].z_offset))) != EEPROM_EMPTY_VALUE16);
 }
 
+
 bool __attribute__((noinline)) eeprom_is_initialized_block(const void *__p, size_t __n) {
-    const uint8_t *top = (const uint8_t*)__p + __n;
-    for (const uint8_t *p = (const uint8_t *)__p; p < top; p++) {
-        if (eeprom_read_byte(p) != EEPROM_EMPTY_VALUE)
+    const uint8_t *p = (const uint8_t*)__p;
```

Comment:
Are we using a block of data bigger than 255 bytes? If not, we could limit `size_t` down to `uint8_t`. But a good check would have been necessary (I don't know how to do that nicely).

Reply:
This only results in 22B of savings